### PR TITLE
Various updates to blocked numbers cmdlet

### PR DESCRIPTION
**Pull request by JensTrier**
Various updates to blocked numbers cmdlet.

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md [hunk 1]
```diff
@@ -38,14 +44,21 @@ Get-CsInboundExemptNumberPattern -Identity "Exempt1"
 
 This returns the exempt number patterns with Identity Exempt1.
 
+### Example 3
+```powershell
+Get-CsInboundExemptNumberPattern -Filter "Exempt*"
+```
+
+This returns the exempt number patterns with Identity starting with Exempt.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md:52]:
```suggestion
This example returns the exempt number patterns with Identity starting with Exempt.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the minor suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the minor suggested change. Many thanks.

@JohanFreelancer9 thanks & done!

File: skype/skype-ps/skype/Remove-CsInboundExemptNumberPattern.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ Removes a number pattern exempt from call blocking.
 ## SYNTAX
 
 ```
-Remove-CsInboundExemptNumberPattern -Identity <XdsGlobalRelativeIdentity> [-Tenant <Guid>] [<CommonParameters>]
+Remove-CsInboundBlockedNumberPattern [-Identity] <string> [-WhatIf] [-Confirm] [<CommonParameters>]
```
Comment (dariomws) [inline skype/skype-ps/skype/Remove-CsInboundExemptNumberPattern.md:20]:
```suggestion
Remove-CsInboundExemptNumberPattern [-Identity] <string> [-WhatIf] [-Confirm] [<CommonParameters>]
```

I think there is a typo here.

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by get-itips — APPROVED** [review]
I am ok once @dariomws suggestion is applied

**Review by JohanFreelancer9 — APPROVED** [review]

File: skype/skype-ps/skype/Remove-CsInboundExemptNumberPattern.md (hunk 2 above)
Comment (JensTrier) [reply skype/skype-ps/skype/Remove-CsInboundExemptNumberPattern.md]: > I think there is a typo here.

@dariomws Thanks!

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsInboundBlockedNumberPattern.md [hunk 3]
```diff
@@ -39,12 +37,20 @@ This cmdlet returns a list of all blocked number patterns added to the tenant li
 PS C:> Get-CsInboundBlockedNumberPattern
```
Comment (tseward) [inline skype/skype-ps/skype/Get-CsInboundBlockedNumberPattern.md:37]: The prompt should be ```PS>``` per https://docs.microsoft.com/en-us/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7.2#avoid-using-powershell-prompts-in-examples.

File: skype/skype-ps/skype/Get-CsInboundBlockedNumberPattern.md [hunk 4]
```diff
@@ -39,12 +37,20 @@ This cmdlet returns a list of all blocked number patterns added to the tenant li
 PS C:> Get-CsInboundBlockedNumberPattern
 ```
 
-In the preceding example, the *Get-CsInboundBlockedNumberPattern* cmdlet is called without any parameters in order to return all the blocked number patterns.
+In this example, the *Get-CsInboundBlockedNumberPattern* cmdlet is called without any parameters in order to return all the blocked number patterns.
+
+### Example 2
+```powershell
+PS C:> Get-CsInboundBlockedNumberPattern -Filter Block*
```
Comment (tseward) [inline skype/skype-ps/skype/Get-CsInboundBlockedNumberPattern.md:44]: Another where the prompt should be ```PS>```.

File: skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md [hunk 5]
```diff
@@ -38,44 +44,22 @@ Get-CsInboundExemptNumberPattern -Identity "Exempt1"
...
-Aliases:
-
-Required: False
-Position: Named
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
+### Example 3
+```powershell
+Get-CsInboundExemptNumberPattern -Filter "Exempt*"
```
Comment (tseward) [inline skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md:49]: Add ```PS>``` to the example.

**Review by tseward — CHANGES_REQUESTED** [review]

**Comment by yogkumgit** [comment]
@JensTrier

**Comment by JensTrier** [comment]
@tseward thanks! I updated the examples.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**